flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -201,16 +201,14 @@ public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
             case "sentry.dart.flutter":
                 setEventEnvironmentTag(event: event, origin: "flutter", environment: "dart")
             case "sentry.cocoa":
-                setEventEnvironmentTag(event: event, origin: "flutter", environment: "dart")
-            case "sentry.native":
-                setEventEnvironmentTag(event: event, origin: "flutter", environment: "dart")
+                setEventEnvironmentTag(event: event, origin: "ios", environment: "native")
```

Comment:
`native` means ObjC/Swift/C/C++ or whatever supported language on iOS, we don't know the language unless there's a way I'm not aware of, maybe @philipphofmann ?

on Android we know when an event was captured on "Java" or "C"